src/coreclr/src/tools/ReadyToRun.SuperIlc/BuildFolder.cs

```diff
@@ -21,12 +21,16 @@ public class BuildFolder
         {
             "coreclr",
             "clrjit",
-            "mscorrc",
-            "mscorrc.debug",
             "mscordaccore",
             "mscordbi",
         };
 
+        private static string[] s_runtimeWindowsOnlyLibraries =
+        {
+            "mscorrc",
```

Comment:
This has been just modified by #33123, we don't have the mscorrc.debug anymore.

Reply:
Beautiful, thanks for pointing that out! Removed in 18th commit.